src/ch/blinkenlights/android/medialibrary/MediaSchema.java

```diff
@@ -302,6 +303,11 @@ public static void upgradeDatabaseSchema(SQLiteDatabase dbh, int oldVersion) {
 			dbh.execSQL(VIEW_CREATE_SONGS_ALBUMS_ARTISTS_HUGE);
 		}
 
+		if (oldVersion < 20170324 && oldVersion >= 20170120) {
+			// Added a string field to allow for string based preferences
+			dbh.execSQL("DROP TABLE preferences");
```

Comment:
You're right, and it should be easy enough to just alter the table to add the string column.